.vscode/settings.json

```diff
@@ -3,16 +3,7 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
-  "eslint.options": {
-    "extensions": [".js", ".json", ".md", ".mdx", ".ts", ".tsx"]
-  },
-  "eslint.validate": [
-    "javascript",
-    "json",
-    "markdown",
-    "mdx",
-    "typescript",
-    "typescriptreact"
-  ],
-  "eslint.packageManager": "pnpm"
+  "eslint.packageManager": "pnpm",
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
Uses version specified in `package.json` instead of allowing VSCode to set the version (this can cause a lot of headaches)